smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/swiftmodules/AWSCRTTypes.kt

```diff
@@ -0,0 +1,16 @@
+package software.amazon.smithy.swift.codegen.swiftmodules
```

Comment:
Several "types" files below.  All of them have SwiftDeclaration added to all symbols, and use the new helper method to construct the symbols.